demo/src/lib/gtag.js

```diff
@@ -0,0 +1,24 @@
+
+// eslint-disable-next-line no-undef
+const GA_TRACKING_ID = process.env.REACT_APP_GA_TRACKING_ID;
+
+/**
+ * Sends a page_view event to Google Analytics.
+ * @param {string} url
```

Comment:
Please add parameter description

Per https://google.github.io/styleguide/jsguide.html#jsdoc-method-and-function-comments
Method, parameter, and return descriptions (but not types) may be omitted if they are obvious from the rest of the method’s JSDoc or from its signature.

In this case it's not obvious that provided url will be used as a page path. Even more confusing that it's called "url" but the value should be the path part of the url (full page url is not acceptable value), consider renaming to pagePath (might work without a description)